change_log/next/FE-1254.yml

```diff
@@ -0,0 +1 @@
+Component Update: Moved component parts to styled-components
```

Comment:
Maybe if you could improve the description here? which 'parts' etc. doesn't make much sense to me currently.